lib/graphql/input_object_type.rb

```diff
@@ -75,16 +75,14 @@ def coerce_non_null_input(value)
 
       arguments.each do |input_key, input_field_defn|
         field_value = value[input_key]
-        field_value = input_field_defn.type.coerce_input(field_value)
 
-        # Try getting the default value
-        if field_value.nil?
```

Comment:
The case I don't cover at the moment is if `coerce_input` returns `nil` because of a failed coercion we used to use the `default_value`. I'm unsure if that was intended since all tests pass.